application/models/UpdateForm.php

```diff
@@ -241,7 +241,7 @@ public function getChangedFiles($destinationBuild)
      * @return object
      * //TODO $downloadid not used locally
      */
-    public function downloadUpdateFile($downloadid, $tobuild)
+    public function downloadUpdateFile($downloadid=null, $tobuild)
```

Comment:
$downloadid not used locally. My logic was if I set to null it can be removed more easily. But since its the 1st param, its not actually.